### ISSUE TITLE
git COMPONENT?

### ISSUE THREAD

**Issue by yarikoptic** [datalad/datalad-installer#46]
would it be feasible to automate native (not in `conda`) installation of `git` on windows?

**Comment by yarikoptic** [comment]
I was trying to install datalad on a fresh windows mini computer, installed only python 3.10 and then within python prompt - `pip.main(["install", "datalad-installer"])` and went from there to install git-annex, but it needs git on windows to be installed first. So -- no success

**Comment by jwodder** [comment]
@yarikoptic I'm going to need an exact sequence of commands to run in order to install Git on Windows.  [This link](https://git-scm.com/download/win) provides downloads but doesn't say what to do with them.

**Comment by yarikoptic** [comment]
Please investigate that sequence of commands, since I do not know it either. Do you still have a windows VM to use?

**Comment by jwodder** [comment]
@yarikoptic Yes, but everything about using it is a hassle.  Don't we have Windows-users on the DataLad team who would be better suited to this?

**Comment by yarikoptic** [comment]
nope - no native Windows users (un)fortunately and people just "enjoying" either VM or appveyor to fix anything related.
